ballista/core/src/utils.rs

```diff
@@ -62,13 +63,14 @@ use tonic::transport::{Channel, Error, Server};
 
 /// Default session builder using the provided configuration
 pub fn default_session_builder(config: SessionConfig) -> SessionState {
-    SessionState::new_with_config_rt(
-        config,
-        Arc::new(
+    SessionStateBuilder::new()
+        .with_default_features()
```

Comment:
previous method did not have the default_features which seemed necessary to me, please check :)

Reply:
That explains a lot. Thanks for catching that!